packages/geo/src/lib/directions/directions.component.ts

```diff
@@ -435,4 +443,17 @@ export class DirectionsComponent implements OnInit, OnDestroy {
       this.languageService
     );
   }
+
+  onToggleDirectionsControl(toggle: boolean) {
```

Comment:
Proposed changed `toggle` to `isActive`? To me, toggle sounds like an action rather than a state.

Reply:
Done.